examples/wip/basis/app.js

```diff
@@ -1,78 +1,108 @@
 /* global document, URL */
 import {load} from '@loaders.gl/core';
-import {BasisLoader} from '@loaders.gl/basis';
-
-const btn = document.createElement('button');
-document.body.appendChild(btn);
-btn.innerText = 'Load Basis File';
+import {BasisLoader, BasisFormat} from '@loaders.gl/basis';
 
 const canvas = document.createElement('canvas');
 document.body.appendChild(canvas);
+canvas.width = 768;
+canvas.height = 512;
+canvas.style.display = 'block';
 const gl = canvas.getContext('webgl');
 const dxtSupported = !!gl.getExtension('WEBGL_compressed_texture_s3tc');
+const loadOptions = {format: BasisFormat.cTFRGB565};
+
+if (dxtSupported) {
+  loadOptions.format = {
+    alpha: BasisFormat.cTFBC3,
```

Comment:
- Basis options should be in `options.basis.---` subobject.

- Could we just use strings in the external API and internally call `BasisFormat[options.basis.type]`?

- There is a function `setLoaderOptions` to set global defaults: